lib/_http_client.js

```diff
@@ -360,10 +360,18 @@ function socketCloseListener() {
   // NOTE: It's important to get parser here, because it could be freed by
   // the `socketOnData`.
   var parser = socket.parser;
+
+  if (!req.res || req.res.readable) {
+    req.emit('aborted');
+    req.aborted = Date.now();
+  }
+
   req.emit('close');
   if (req.res && req.res.readable) {
     // Socket closed before we emitted 'end' below.
     req.res.emit('aborted');
+    req.aborted = Date.now();
```

Comment:
I think this property is currently only set when calling `req.abort()` so this looks like a semver-major change.